subprojects/snapshots/src/main/java/org/gradle/internal/change/DefaultFileChange.java

```diff
@@ -86,10 +88,15 @@ public File getFile() {
     @Override
     public ChangeType getChangeType() {
         // TODO wolfs: Shall we do something about file type Missing -> File
-        // should this be file type ADDED instead of MODIFIED
+        //   should this be file type ADDED instead of MODIFIED
```

Comment:
Nit: ws.

Reply:
The whitespace is intentional, so Intellij considers this as a two line TODO comment instead of a one line TODO and a separate comment.

Reply:
Interesting, thanks.